boards/arm/frdm_k64f/pinmux.c

```diff
@@ -123,7 +123,7 @@ static int frdm_k64f_pinmux_init(struct device *dev)
 #endif
 
 #if DT_HAS_NODE_STATUS_OKAY(DT_NODELABEL(ftm3)) && \
-    DT_NODE_HAS_COMPAT(DT_NODELABEL(ftm3), nxp_kinetis_ftm_pwm)
+    DT_NODE_HAS_COMPAT_STATUS_OKAY(DT_NODELABEL(ftm3), nxp_kinetis_ftm_pwm)
```

Comment:
Wouldn't it be better to have the `OKAY` as a parameter to the macro? The name is pretty ugly and to top that it's inconsistent with `DT_INST_FOREACH_OKAY`

Reply:
Maybe it's better to make DT_INST_FOREACH_OKAY DT_INST_FOREACH_STATUS_OKAY instead. The STATUS_OKAY thing is being used elsewhere.